doc/how-to/ceph_networking.rst

```diff
@@ -0,0 +1,161 @@
+.. _howto-ceph-networking:
+
+How to configure Ceph networking
+================================
+
+When running :command:`microcloud init`, you are asked if you want to provide custom subnets for the Ceph cluster.
+Here are the questions you will be asked:
+
+- ``What subnet (either IPv4 or IPv6 CIDR notation) would you like your Ceph public traffic on? [default: 203.0.113.0/24]: <answer>``
+- ``What subnet (either IPv4 or IPv6 CIDR notation) would you like your Ceph internal traffic on? [default: 203.0.113.0/24]: <answer>``
+
+You can choose to skip both questions (just hit ``Enter``) and use the default value which is the subnet used for the internal MicroCloud traffic.
+This is referred to as a *usual* Ceph networking setup.
+
+.. figure:: /images/ceph_network_usual_setup.svg
+   :alt: All the Ceph traffic is on the same network interface
```

Comment:
Can you add `:width: 100%` here (and below)?
That *should* make the images clickable.

Reply:
fixed